code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -0,0 +1,23 @@
+/datum/action/changeling/gaseous_pores
+	name = "Gaseous Pores"
+	desc = "Our skins bursts, releasing noxious gases to confuse opponents and cover our retreat."
+	helptext = "We are still affected by the gases we emit, internals are recommended.Functions while unconscious."
+	button_icon_state = "smoke"
+	chemical_cost = 35
+	dna_cost = 2
+	req_stat = UNCONSCIOUS
```

Comment:
Did you test it? Does it allow to release it even if you are conscious?

Reply:
Yes, it does